winpr/include/winpr/wtypes.h

```diff
@@ -49,17 +57,39 @@
 #define NEAR
 #endif
 
+#if defined(HAVE_STDINT_H)
```

Comment:
I'm not sure to understand, I was meaning that here it should be
#if not defined(HAVE_STDINT_H)